.github/workflows/pr-checklist.yml

```
@@ -8,4 +8,8 @@ jobs:
   pr-checklist:
     runs-on: ubuntu-latest
     steps:
-      - uses: it-at-m/lhm_actions/action-templates/actions/action-pr-checklist@14080db5171a3b2a879ea09e18ea5a276fbfb189 # v1.0.1
\ No newline at end of file
+      - name: Dump GitHub context
+        env:
+          GITHUB_ACTOR: ${{ github.actor }}
+        run: echo "Actor in workflow $GITHUB_ACTOR"
+      - uses: it-at-m/lhm_actions/action-templates/actions/action-pr-checklist@ead3fd86af279d3639f6de012354904871f6be0e
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)